src/python/CRABInterface/RESTWorkerWorkflow.py

```diff
@@ -10,7 +10,7 @@
 from CRABInterface.Utilities import getDBinstance
 from CRABInterface.RESTExtensions import authz_login_valid
 from CRABInterface.Regexps import (RX_TASKNAME, RX_WORKER_NAME, RX_STATUS, RX_TEXT_FAIL, RX_SUBPOSTWORKER,
-                                  RX_SUBGETWORKER, RX_JOBID)
+                                  RX_JOBID)
```

Comment:
why remove SUBGETWORKER but not SUBPOSTWORKER and what has this to do with panda ?
Can you elaborate a  bit on the rational ?

Reply:
Your commit https://github.com/belforte/CRABServer/commit/5a8fa719c707f58a5cfa688e3eb288c042bedc5d#diff-f457a331093710562f796332dd9f5ba55f470f9c1adf1adb4048b1c7e32e20f9
removed `RX_SUBGETWORKER` from `CRABInterface.Regexps`, i could not do anything but removing the import and not using it

Reply:
thanks Dario. Sorry I did not look. I think I understand now. Well... things could have been named better to begin with. Let me review things in detail. We do not use the `jobgroup` table in DB anymore.

Reply:
OK. indeed `subresource` is not used for get and you should kindly remove `subresource` from L90 to make it clear.

Reply:
this is a cleanup which could have been done long ago :-(